logic/src/commonMain/kotlin/com/wire/kalium/logic/data/user/UserRepository.kt

```diff
@@ -647,6 +651,18 @@ internal class UserDataSource internal constructor(
         userDAO.getNameAndHandle(userId.toDao())
     }.map { NameAndHandle.fromEntity(it) }
 
+    override suspend fun migrateUserToTeam(teamName: String): Either<CoreFailure, CreateUserTeamDTO> {
```

Comment:
We should not expose DTO data classes to higher level classes(Eg. usecases). We can map it here to `CreateUserTeam`